### DISCUSSION TITLE
how to install COMDIRAC

### DISCUSSION THREAD

**Discussion by adriansev** [DIRACGrid/DIRAC#6650] [Q&A] [answered]
Hi! How can i install COMDIRAC within the new DIRACOS based installation?
Thanks a lot!

**Comment by adriansev** [comment]
i found that i can do `python3 -m pip install COMDIRAC` but it did uninstall my latest version of DIRAC
```
Installing collected packages: DIRAC, COMDIRAC
  Attempting uninstall: DIRAC
    Found existing installation: DIRAC 8.0.8
    Uninstalling DIRAC-8.0.8:
      Successfully uninstalled DIRAC-8.0.8
Successfully installed COMDIRAC-1.3.3 DIRAC-7.3.34
```

Is this something that can be fixed?

**Comment by marianne013** [comment]
Not that this will help you right now, but we are trying to get COMDIRAC merged into main DIRAC which should avoid future issues like this:
https://github.com/DIRACGrid/DIRAC/pull/6403

**Comment by chrisburr** [comment, marked as answer]
COMDIRAC is now included from v8.0.14 onwards.

**Reply by adriansev** [reply]
Thanks a lot!
